controllers/graph.controller.js

```diff
@@ -0,0 +1,84 @@
+const graphDbService = require("../services/graph.service");
+const {
+  ERROR_GRAPH_NOT_FOUND,
+  ERROR_INVALID_GRAPH,
+  INTERNAL_ERROR,
+} = require("../constants");
+const { isValidGraph } = require("../services/utils");
+
+const publishGraph = async (req, res) => {
+  try {
+    const { adjacencyLists, userId } = req.body;
+    if (!isValidGraph(adjacencyLists)) {
+      res.send({ error: ERROR_INVALID_GRAPH });
+      return;
+    }
+    const newGraph = await graphDbService.DbCreateGraph({
+      adjacencyLists,
+      userId,
```

Comment:
Also, you can change your method naming into `createGraph` - no need for the Db part. 

It will be just as understandable